spec/factories/shopping_cart_items.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+ factory :shopping_cart_item do
+   quantity 1
+   price_cents 25
+   price_currency "SEK"
+ end
```

Comment:
I could be wrong, but I don't think there's a use case for a `shopping_cart_item` factory. When would we need to do this? We always create shopping cart items by adding a dish to a cart, right? We never need to manually create an "item"